torchci/pages/api/drci/drci.ts

```diff
@@ -823,6 +848,27 @@ export async function getWorkflowJobsStatuses(
         }
       }
 
+      if (
+        matchDisabledTestIssues !== undefined &&
+        matchDisabledTestIssues.length !== 0 &&
+        isDisabledTest(matchDisabledTestIssues)
+      ) {
+        if (isDisabledTestMentionedInPR(matchDisabledTestIssues, prInfo)) {
+          // If the test is disabled and it's mentioned in the PR, its failure
+          // would be legit, for example, the PR is trying to fix the flaky test
+          relatedIssues.set(job.id, matchDisabledTestIssues);
+        } else {
+          // If the test is disabled and it's NOT mentioned anywhere in the PR,
+          // its failure is consider flaky
+          flakyJobs.push(job);
+          const disabledTestIssuesMsg = matchDisabledTestIssues
+            .map((issue) => `[#${issue.number}](${issue.html_url})`)
+            .join(", ");
+          relatedInfo.set(job.id, `disabled by ${disabledTestIssuesMsg}`);
+          continue;
+        }
+      }
+
```

Comment:
Does there happen to be a PR out there that attempts to fix a test issue but it still fails on the PR?  That way you can test this path more

Reply:
Let me create a mock one to test it out.  That would be an easy thing to do, just take a bit more time.

Reply:
I have tested that out here https://github.com/pytorch/pytorch/pull/125634 by reusing Alban empty PR description.  If the issue is mentioned, the failed test is treated as a new failure with the link to the disabled issue provided as additional information.